GiveNGo/GiveNGo
# Dashboard page

## Description:
- This change adds a basic layout to the dashboard page. The dashboard page renders one task card and one request card. I will be adding functionality for more tasks and requests to be displayed in a future PR. I also fixed an issue with the React Native version not matching.

## Changes I Made:
- [X] In DashboardPage.tsx, add text and dividers for current tasks and current requests. This container renders a task card and a request card.
- [X] Create TaskCard.tsx. This component renders a simple card with text on it.
- [X] Create RequestCard.tsx. This component renders a simple card with text on it.
- [X] In package.json, update dependencies and devDependencies for React Native to both use version 0.62.2.

## How to Test:
1. npm install
2. expo start
3. Click to login and accept. 
4. This will navigate you to the dashboard page.
5. The dashboard page should look like the screenshot below. 

![DashboardPR7](https://user-images.githubusercontent.com/7851187/88221231-03e7e800-cc19-11ea-9935-a76fa4aa5676.png)
